blueprint/security/hybrid-system-security-plan.md

```diff
@@ -821,20 +841,28 @@ The HybridSystem leverages Azure AD for controlling system access. All technical
+The HybridSystem provides security controls and an identity management framework that can be utilised to manage system access for systems deployed within the HybridSystem. The HybridSystem enforces multi-factor authentication through Conditional Access policies, creates recovery (emergency access) accounts for maintaining access to resources and enforces password policies for accounts created directly in Azure AD. The HybridSystem uses a soft-token - the Microsoft Authenticator app - to reduce the need for purchase, distribution and management of hard-tokens.
+
+The Microsoft Authenticator app cannot be centrally managed to require the enforcement of a PIN or biometric unlock, and is not considered verifier impersonation resistant.
+
+The Protected User security group is not available in Azure AD and is therefore not used by the HybridSystem.
 
 The HybridSystem utilises Azure AD to store groups utilised for RBAC and provides process and administration documentation for managing access to Azure resources.
 
+To minimise potential user impact, Windows 10 laptops are not rebooted daily. Instead, automatic reboots are only performed as part of automated patching.
+
 #### Security controls provided by the HybridSystem
 
 * Azure AD is configured to require all users to be authenticated before granting access.
-* Azure MFA is enforced for all standard and privileged users.
-* MFA requires complex password and One Time Password (OTP) from Microsoft Authenticator App (soft token).
-* None of the authentication factors on their own can be used for single-factor authentication to another system.
-* Azure MFA is enforced for all users accessing Office 365 content.
-* Azure AD self-service password reset requires users to verify their identity before resetting their password in accordance with password complexity requirements.
-* Local Area Network (LAN) Manager is not used by the HybridSystem, however New Technology LAN Manager (NTLM) is used within the system.
-* Credentials are stored within Azure AD. Azure AD Identity Protection is enabled to detected leaked passwords.
+* Azure MFA is enforced for all standard and privileged users accessing M365 services.
+* MFA requires Azure AD password and Microsoft Authenticator app (either acceptance of push notification or entry of OTP.
+* Azure AD password complexity enforces a minimum character length of 8 characters.
+* All authentication attempts are logged in Azure AD Sign-ins.
+* Azure AD logs are forwarded to a Log Analytics workspace for long-term secure retention.
+* Legacy authentication methods are disabled in the Windows 10 SOE following the ACSC hardening guide.
+* Credentials are stored within Azure AD and the on-premises AD.
+* Standard Windows & iOS functionality is to obscure passwords during logon.
+* Windows Defender Credential Guard is enabled for the Windows 10 SOE. 
```

Comment:
should it be mentioned if the device supports it or is this assumed?

Reply:
The hardware requirements in the Client Devices design specify what is required to support Credential Guard so I think we can take it as assumed here.